vision/segmentation/detections_to_objects.go

```diff
@@ -35,6 +35,30 @@ func (dsc *DetectionSegmenterConfig) ConvertAttributes(am utils.AttributeMap) er
 	return decoder.Decode(am)
 }
 
+func cameraToProjector(
+	ctx context.Context,
+	source camera.VideoSource,
+) (transform.Projector, error) {
+	if source == nil {
+		return nil, errors.New("cannot have a nil source")
+	}
+	props, err := source.Properties(ctx)
+	if err != nil {
+		return nil, camera.NewPropertiesError("source camera")
+	}
+	if props.IntrinsicParams == nil {
+		return &transform.ParallelProjection{}, nil
+	}
```

Comment:
Edited the code to add a default ParallelProjector here

Reply:
Thank you so so so much!